scripts/xe-reset-networking

```diff
@@ -67,6 +67,7 @@ if __name__ == "__main__":
 	parser.add_option("--netmask", help="Netmask for new management interface", dest="netmask", default='')
 	parser.add_option("--gateway", help="Gateway for new management interface", dest="gateway", default='')
 	parser.add_option("--dns", help="DNS server for new management interface", dest="dns", default='')
+	parser.add_option("--use_ipv6", help="Network will be reconfigured with IPv6", dest="use_ipv6", action="store_const", const=True, default=False)
```

Comment:
Small thing but I would prefer "Reconfigure network .." to express that this is a command.